src/main/webapp/js/components/fields/field-config-modal.jsx

```diff
@@ -243,6 +246,7 @@ class FieldConfigModal extends React.Component {
 
 const mapStateToProps = (state, props) => ({
   fieldConfig: getFieldConfigById(state, props),
+  entities: state.fieldConfig[props.entityType],
```

Comment:
I think you can rename the entities to configs or fieldConfigs to be more descriptive